File: BlazorApp1/Estabelecimentos.Modelos/Funcionario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ControleDeEstabelecimentos.Modelos
{
    public class Funcionario
    {
        protected int id;
        protected string nome;
        protected string email;
        protected string telefone;
        public int Id { get { return id; }  set { id = value; } }
        public string Nome { get { return nome; } set { nome = value; } }
        public string Email { get { return email; } set { email = value; } }
        public string Telefone { get { return telefone; } set { telefone = value; } }

        // Metodos
        protected void IniciarFila(int Id)
        {

        }
        protected void FinalizarFila(int Id)
        {

        }
        protected string ChamarProximo(int Id, string NumeroFila)
        {
            string ProximoNumero = NumeroFila;
            return ProximoNumero;
        }


    }
}

File: PHP/Controller/Funcionario.php
<?php
public class Funcionario
{
    
    // Atributos
    protected int $id;
    protected string $nome;
    private string $email;
    protected int $telefone;

    //Getters
    public function getId() 
    {
         return $this->id;
    }
    public function getNome() 
    {
         return $this->nome;
    }
    public function getEmail() 
    {
         return $this->email;
    }
    public function getTelefone() 
    {
         return $this->telefone;
    }

    //Setters
    public function setId($Id) 
    {
        $this->id = $Id;
    }
    public function setNome($Nome) 
    {
        $this->nome = $Nome;
    }
    public function setEmail($Email) 
    {
        $this->email = $Email;
    }
    public function setTelefone($Telefone) 
    {
        $this->telefone = $Telefone;
    }


    // Metodos
    public void AbrirFila(int Id)
    {

    }
    public void FecharFila(int Id)
    {

    }
    public string ChamarProximo(int Id, string NumeroFila)
    {
        string ProximoNumero = NumeroFila;
        return ProximoNumero;
    }


}
?>